test/heap.json

```diff
@@ -18983,3 +18983,4 @@
 {"address":"0x7f83e10bbf50", "type":"OBJECT", "class":"0x7f83e1046110", "ivars":63, "references":["0x7f83e10b9b38", "0x7f83e10b9c50", "0x7f83e10b9ca0", "0x7f83e10ba768", "0x7f83df927ab0", "0x7f83df8340e0", "0x7f83e10bb6e0", "0x7f83df834018", "0x7f83df8342e8", "0x7f83e10bb258", "0x7f83df9278a8", "0x7f83e10bb8c0", "0x7f83e10bbf00", "0x7f83e10bbed8", "0x7f83e10bb708", "0x7f83e10bbe88", "0x7f83e10bbe60", "0x7f83e10bbe38", "0x7f83e10bbe10", "0x7f83e10bbde8", "0x7f83e10bb6b8", "0x7f83e10bbd98", "0x7f83e10bbd70", "0x7f83e10bbd48", "0x7f83e10bb988", "0x7f83e10bb690", "0x7f83e10bbb68", "0x7f83e10bbc80", "0x7f83de80a200", "0x7f83e10bbc30", "0x7f83e0813790", "0x7f83e10b9bd8", "0x7f83e10b9db8"], "memsize":544, "flags":{"wb_protected":true, "old":true, "uncollectible":true, "marked":true}}
 {"address":"0x7f83e10bbf78", "type":"IMEMO", "imemo_type":"memo", "references":["0x7f83df8cfc48", "0x7f83e10bbfa0"], "memsize":40, "flags":{"wb_protected":true}}
 {"address":"0x7f83e10bbfa0", "type":"ARRAY", "class":"0x7f83df8b4088", "length":1, "embedded":true, "references":["0x7f83df8005d8"], "memsize":40, "flags":{"wb_protected":true}}
+{"address":"0x7f83e10bbff0", "type":"STRING", "class":"0x7f83df8cfc48", "embedded":true, "bytesize":3, "value":"���", "memsize":40, "flags":{"wb_protected":true}}
```

Comment:
I hex-edited the value to be FF FF FF

Reply:
This should probably be it's own test. A bit not great to throw it in here at the end. I'll fix it.

Reply:
done